src/sidebar/components/ShareDialog/test/ShareDialog-test.js

```diff
@@ -79,20 +77,10 @@ describe('ShareDialog', () => {
     assert.isFalse(wrapper.find('TabHeader').exists());
   });
 
-  [
-    [{ shareTab: false }],
-    [{ importTab: false }],
-    [{ exportTab: false }],
-    [{}],
-  ].forEach(props => {
+  [{ importTab: false }, { exportTab: false }, {}].forEach(props => {
     it(`renders a tabbed dialog when more than one tab is provided`, () => {
       const wrapper = createComponent(props);
-
       assert.isTrue(wrapper.find('TabHeader').exists());
-      assert.isTrue(
-        wrapper.find('Tab[aria-controls="share-panel"]').props().selected,
-      );
-      assert.isTrue(wrapper.find('TabPanel[id="share-panel"]').props().active);
```

Comment:
These tests were already wrong, but that was made obvious by the issue explained here https://github.com/hypothesis/client/pull/5876/files#r1341383609